### DISCUSSION TITLE
ariaLabelOverride not working

### DISCUSSION THREAD

**Discussion by Alison-Sarah-P-C** [ritikbanger/react18-input-otp#40] [Q&A] [answered]
- The `ariaLabelOverride` property is not working
- Requested enhancement: ability to use the `index` within a custom aria label through the override/new callback to get the current `index` that can then be used however needed

The property `ariaLabelOverride` is not working. I have managed to get a workaround using this:
`inputProps={{ "aria-label": "Custom aria label" }}`

But it would be great to get the dedicated parameter working. 

Also what would be _amazing_ is if there was a way to get access to the `index` to be able to add it into the aria label. The default aria label says which digit you are on for each input which is a better experience for users that require a screen reader, would be great to be able to add a custom aria label (we  translate it based on locale) but still have that index information included.

**Comment by ritikbanger** [comment]
@Alison-Sarah-P-C Could you share me your implementation as well as the DOM screenshot to check if it is not working.

**Comment by ritikbanger** [comment]
@Alison-Sarah-P-C Any update on your implementation, this issue would be closed due to non-response after 7 days.

**Comment by ritikbanger** [comment, marked as answer]
`inputProps={{ "aria-label": "Custom aria label" }}` is valid
